### PR TITLE
chore: Add sentry tracing package 

### PR DESCRIPTION
**Pull request by shruthilayaj**
Initialize sentry right after express app is initialized, add tracing handler
Related [PR](https://github.com/getsentry/sentry/pull/28114)

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -11,7 +10,6 @@ import {renderStream} from './renderStream';
 import {PollingConfig} from './types';
 
 dotenv.config();
-Sentry.init({dsn: process.env.SENTRY_DSN});
```
Comment (EvanPurkhiser) [inline src/index.ts:14]: Damn, I guess we'll only have sentry for the server version huh 😬

Comment (shruthilayaj) [reply src/index.ts:14]: I don't fully understand how it works, but yeah I had to add the [express integration](https://github.com/getsentry/chartcuterie/pull/54/files#diff-4a3797637f93391dd49a34160aec29d8a0390c214ca2e2ef1ccdf284c35f6998R21) with the `app`

`new Integrations.Express({app})`

Comment (EvanPurkhiser) [reply src/index.ts:14]: I wonder if we can add the integration retroactively after the inital `Sentry.init`

Comment (shruthilayaj) [reply src/index.ts:14]: I don't know enough about node.js to quickly figure out if we can do this 😬  do you have any suggestions?

Comment (EvanPurkhiser) [reply src/index.ts:14]: https://forum.sentry.io/t/add-integration-after-init/5260/4 here we go

**Review by EvanPurkhiser — APPROVED** [review]

**Pull request merged**